routing/src/user/userController.ts

```diff
@@ -0,0 +1,17 @@
+import { addUser, getUser, updateUser, deleteUser } from './userService';
+
+export function addUserController(data: string): number {
```

Comment:
Allow the **request** and **response** parameters in controller and return appropriate response with status code.